tracy-client/tests/tests.rs

```diff
@@ -0,0 +1,91 @@
+use tracy_client::*;
+
+#[global_allocator]
+static GLOBAL: ProfiledAllocator<std::alloc::System> =
+    ProfiledAllocator::new(std::alloc::System, 100);
+
+fn basic_zone() {
+    let client = Client::enable();
+    let span = client.span(srcloc!("basic_zone"), 100);
```

Comment:
I think this is a big step backwards in usability. Compare: `span!("basic_zone")`.

Reply:
It sure is. It wouldn't be a huge problem to add the macro back in, although it would have to call a `Client::enable` for each invocation. I guess I could also make it optionally take the client as an argument too.

Reply:
An alternative approach would be to track enabled-ness globally with an `AtomicBool`; then nobody has to deal with `Client`s and the existing API can be preserved.